kumascript/macros/EmbedInteractiveExample.ejs

```diff
@@ -10,14 +10,6 @@
 
 let iClass = "interactive";
 let url = kuma.url.resolve(env.interactive_examples.base_url, $0);
-let siteURL = mdn.siteURL();
-if (siteURL != 'https://developer.mozilla.org') {
-    if (url.includes('?')) {
-        url += '&origin=' + siteURL;
-    } else {
-        url += '?origin=' + siteURL;
-    }
-}
```

Comment:
Even if the `origin` query parameter may be used by the interactive-examples service, it never reaches it, because the interactive-examples CDN strips all query parameters before forwarding the request to the interactive-examples S3 bucket. So, no harm in removing this code.